### PR TITLE
Improve OCP job error handling and increase job cpu and memory

### PR DESCRIPTION
**Pull request by awcrosby**
Works https://github.com/ansible/galaxy_ng/issues/64

### CODE REVIEW

File: galaxy_importer/ansible_test/job_template.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
         image: {image}
         resources:
           limits:
```
Comment (cutwater) [inline galaxy_importer/ansible_test/job_template.yaml]: Please add `requests:` to resource section.

Comment (awcrosby) [reply galaxy_importer/ansible_test/job_template.yaml]: @cutwater I will probably set the request at:
```
requests:
  cpu: 250m
  memory: 256Mi
```

Comment (awcrosby) [reply galaxy_importer/ansible_test/job_template.yaml]: Had to adjust to `500m` and `512Mi` accommodate larger collections

File: galaxy_importer/ansible_test/runners/openshift_job.py [hunk 2]
```diff
@@ -281,7 +281,11 @@ def get_pods(self):
         params = {'labelSelector': f'job-name={self.name}'}
         r = requests.get(
             self.pods_url, headers=self.auth_header, params=params, verify=self.ca_path)
-        return r.json()['items']
+        try:
+            pods = r.json()['items']
+        except KeyError:
```
Comment (cutwater) [inline galaxy_importer/ansible_test/runners/openshift_job.py:286]: `r.json()` can also raise ValueError if HTTP response body is not a valid JSON.

Comment (awcrosby) [reply galaxy_importer/ansible_test/runners/openshift_job.py]: @cutwater updated

**Pull request merged**